test/plotting.jl

```diff
@@ -0,0 +1,29 @@
+@testset "plotting" begin
+    possible_inclusivities = Iterators.product((true, false), (true, false))
+    @testset "Interval{Float64} with inclusivity=$inc" for inc in possible_inclusivities
+        intervals = [Interval(float(x), float(x + 0.5), inc...) for x in 1:11]
+        @plottest plot(intervals, 1:11) "references/interval_$inc.png" false
```

Comment:
The file names for these are kind of ridiculous. This is probably a better alternative: 
```suggestion
        @plottest plot(intervals, 1:11) "references/interval_$(join(inc ? 'T' : 'F' for inc in incs)).png" false
```

Reply:
I don't see a huge problem with `interval_(false, false).png`
and I find
`@plottest plot(intervals, 1:11) "references/interval_$inc.png" false`
much easier to read than:
`@plottest plot(intervals, 1:11) "references/interval_$(join(inc ? 'T' : 'F' for inc in incs)).png" false`

So I am inclined to leave as is.
Not like anyone interacts with these files directly.
Noone ever has to type that filename

Reply:
That's easy enough to solve:
```julia
filename = "interval_" * join(inc ? 'T' : 'F' for inc in incs) * ".png"
@plottest plot(intervals, 1:11) "references/$filename" false
```
I still have to look at these file names when maintaining the repo so it be good to make them less ugly :P

Reply:
How about:
```suggestion
        @plottest plot(intervals, 1:11) "references/interval_$(join(inc)).png" false
```

So the filenames would be along the lines of `"references/interval_truefalse.png"`

Reply:
I'm good with that. Also bump the package version and we can get this merged

Reply:
done